<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountWindow</class>
 <widget class="QWidget" name="accountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Licensing</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="gui_quit">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gui_upgradeLicense">
      <property name="text">
       <string>Change License</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>281</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="gui_username">
      <property name="text">
       <string>Username: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="gui_licenseState">
      <property name="text">
       <string>License Type:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="gui_licenseSelector">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No License</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Standard License</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Premium License</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
